lib/src/operance_data_column_header.dart

```diff
@@ -255,65 +268,69 @@ class _ColumnHeader<T> extends StatelessWidget {
   /// Indicates whether the column is being dragged.
   final bool dragging;
 
+  final double columnWidth;
+
   @override
   Widget build(BuildContext context) {
     final colors = decoration.colors;
     final icons = decoration.icons;
     final styles = decoration.styles;
 
-    return Container(
-      alignment: Alignment.centerLeft,
-      padding: EdgeInsets.symmetric(horizontal: 8.0),
-      decoration: styles.columnHeaderDecoration.copyWith(
-        color: dragging
-            ? styles.columnHeaderDecoration.color
-            : styles.columnHeaderDecoration.color!.withOpacity(0.5),
-      ),
-      width: column.width.value(tableWidth),
-      height: decoration.sizes.columnHeaderHeight,
-      child: Row(
-        children: <Widget>[
-          column.columnHeader,
-          decoration.ui.columnHeaderSortIconSpaced
-              ? const Spacer()
-              : const SizedBox(width: 12.0),
-          if (column.sortable)
-            Builder(
-              builder: (context) {
-                final columnName = column.name;
-                final sortIndex = sorts.keys.toList().indexOf(columnName);
-                final direction =
-                    sortIndex != -1 ? sorts.values.toList()[sortIndex] : null;
-
-                return MouseRegion(
-                  cursor: SystemMouseCursors.click,
-                  child: GestureDetector(
-                    onTap: onSort != null
-                        ? () {
-                            onSort!(
-                              columnName,
-                              sortIndex == -1
-                                  ? SortDirection.ascending
-                                  : direction == SortDirection.ascending
-                                      ? SortDirection.descending
-                                      : null,
-                            );
-                          }
-                        : null,
-                    child: Icon(
-                      direction == SortDirection.ascending
-                          ? icons.columnHeaderSortAscendingIcon
-                          : icons.columnHeaderSortDescendingIcon,
-                      color: sortIndex != -1
-                          ? colors.columnHeaderSortIconEnabledColor
-                          : colors.columnHeaderSortIconDisabledColor,
-                      size: decoration.sizes.columnHeaderSortIconSize,
-                    ),
+    final sortIconBuilder = column.sortable
+        ? Builder(
```

Comment:
this can just be placed in the widget below no need for a variable if it's only being used in 1 place, the condition below can just 
        child: Row(
          children: <Widget>[
            Expanded(child: column.columnHeader),
            if (column.sortable)
               Builder(...)
          ],
        ),